src/_includes/feature_lists/cloud.njk

```diff
@@ -104,7 +143,12 @@
         </li>
         {% for row in section.rows %}
         <li class="ff-feature-row">
-            <label>{{ row.label }}</label>
+            <label>
+                {{ row.label }}
+                {% if row.info %}
+                <i id="ff-info--{{ row.id }}" class="ff-icon" onclick="openInfo('{{ row.label }}')">{% include "components/icons/info.svg" %}</i>
```

Comment:
Why onclick and not a hover action?

Reply:
Given the amount of content that could exist for each feature, a dialog/modal works better. You don't show/hide modals via hover - only really tooltips.